app/views/routes/show.html.erb

```diff
@@ -5,5 +5,31 @@
   <%= @route.name %>
 </p>
 
+<p>
+  <strong>Trains:</strong>
+  <ul>
+  	<% if @route.trains.present? %>
+  		<% @route.trains.each do |train| %>
+  			<li><%= train.number %></li>
+  		<% end %>
+  	<% else %>
+  		There are no trains on the route
+  	<% end %>
+  </ul>
+</p>
+
+<p>
+  <strong>Stations:</strong>
+  <ul>
+  	<% if @route.railway_stations.present? %>
```

Comment:
Are this indents correct ? There are looks enormous.

Reply:
Why?